plugins/chat/assets/javascripts/discourse/components/chat/navbar/new-channel-button.gjs

```diff
@@ -22,10 +22,7 @@ export default class ChatNavbarNewChannelButton extends Component {
         @action={{this.createChannel}}
         @icon="plus"
         @label={{if this.site.desktopView "chat.create_channel.title"}}
-        class={{concatClass
-          "c-navbar__new-channel-button"
-          (if this.site.mobileView "btn-flat")
-        }}
+        class={{concatClass "c-navbar__new-channel-button btn-transparent"}}
```

Comment:
```suggestion
        class="c-navbar__new-channel-button btn-transparent"
```

Reply:
That might cause a linting error if concatClass is used nowhere now (you will need to remove the import in this case)